RELEASE_NOTES.md

```diff
@@ -2,6 +2,65 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 3.0.0
+
+This is a performance improvement release, focused on memory utilization.
+The improvements introduce some backwards breaking changes which are covered in detail below.
+
+### Performance Improvements
+
+Refinery's memory usage improvements are the result of reducing the number data transformations performed during the receipt and retransmit of traces.
+Reduced data transformations has reduced memory allocations and time spent in garbage collection.
+To take full advantage of these optimizations, send telemetry to Refinery in supported binary formats whenever possible.
+When sending OTLP, use http/protobuf (preferred) or GRPC.
+If you are sending OTLP as http/json, consider using an OpenTelemetry Collector to receive it and send to Refinery with an OTLP exporter using http/protobuf (preferred) or GRPC.
+When sending libhoney events (using Beelines), use msgpack.
+
+### Configuration Options (💣 Breaking Changes!)
+
+You may need to update your Refinery configuration or the cluster will fail to start or behave as intended.
+
+* Removed:
+  * Trace Redistribution
+    * `DisableRedistribution` - **Remove** this option. The redistribution of in-progress traces during Refinery scaling events is no longer supported.
+    * `RedistributionDelay` - **Remove** this option. Redistribution is no longer supported.
+  * Outgoing Buffer Sizes
+    * `PeerBufferSize` and `UpstreamBufferSize` - **Remove** these options. These queue sizes no longer need to be set.
+  * Trace Locality Mode (experimental)
+    * `TraceLocalityMode` - **Remove** this option. This experimental feature has been removed.
+    * `MaxDropDecisionBatchSize`, `DropDecisionSendInterval`, `MaxKeptDecisionBatchSize`, and `KeptDecisionSendInterval` - Remove these options related to `TraceLocalityMode`.
+
+* Replace:
+  * `CacheCapacity` - **Remove** this option.
+    * **Replace** it with `IncomingQueueSize` and `PeerQueueSize` if these are not already set.
+    * Appropriate starting values for these queue sizes are 3 times the previous value of `CacheCapacity`.
+  * `LegacyMetrics` - **Replace** with `OTelMetrics`
+    * LegacyMetrics was deprecated in v2.0.0 and is being removed in this release.
+    * Both metric types have the same configuration options.
+    * Some metrics will have slightly different names after this change; see below for details.
+
+### Metrics (️⚠️ Breaking Changes!)
+
+Some metrics have been removed, some added, some renamed, some behave slightly differently.
+These changes mean that existing boards or alerts using the affected metrics will break and require updating.
+
+* Removed:
```

Comment:
Are there any recommended alternatives to these removed metrics?

Reply:
Maybe this is a good reason for adding the newly introduced metrics in the release note as well